app/views/paths/index.html.erb

```diff
@@ -49,8 +56,15 @@
     <div class="flex gap-x-10 gap-y-6 flex-col md:flex-row">
       <% @fullstack_paths.each do |path| %>
 
-        <%= render CardComponent.new(classes: 'flex flex-col') do |card| %>
+        <%= render CardComponent.new(classes: 'flex flex-col relative') do |card| %>
           <% card.with_header(classes: 'flex flex-col justify-between items-center') do %>
+            <div class="absolute top-3.5 right-3.5">
+              <% if current_user.present? and current_user.on_path?(path) %>
```

Comment:
```suggestion
              <% if current_user.present? && current_user.on_path?(path) %>
```
Nit, but for the sake of consistency (both with your code above and with this codebase in general), I think it's better to use `&&`.